### PR TITLE
Add Desktop version to the portfolio

### PR DESCRIPTION
**Pull request by aubinleyoung**
I have added a desktop version to the portfolio using media queries to adapt it to the desktop screen

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -356,3 +357,238 @@
...
+    width: 600px;
+    height: auto;
+  }
+
+  .recent-works {
+    grid-column: 1/4;
+    grid-row: 2/3;
+    display: grid;
+    grid-template-columns: auto auto;
+  }
```
Comment (Tadesse-Alemayehu) [inline style.css:464]: - [ ] Dear @aubinleyoung, Although your feature project is looking good, One thing missing is you need to align the image and the text properly. Please make sure The featured project image takes 60% of the page's width and is aligned to the left, and the description takes the remaining 40% and is aligned to the right. 

### Figma design 
sdfsdf

### Your dsign
Capture

File: style.css [hunk 2]
```diff
@@ -87,7 +87,7 @@
 /* today */
 #works-container {
   display: grid;
-  grid-template-rows: 150px auto 450px 450px 450px 100px;
+  grid-template-rows: 150px auto 450px 450px 450px 450px 450px 450px 100px;
   padding-top: 8em;
 }
```
Comment (Tadesse-Alemayehu) [inline style.css:92]: - [ ] Impressive styling for your works. Also, the missing part here is the `hover` effect. Please make sure there is a `see project` button that appears on hover for each project. when the project is hovered on with the mouse the description and dark opacity disappear and a `See Project` button appears to the bottom of the image and takes the whole image width.

### Figma design `on hover`
card-hover

### Your design
card

**Review by Tadesse-Alemayehu — CHANGES_REQUESTED** [review]
### Hi @aubinleyoung 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
EquatorialVictoriousBlackpanther-size_restricted

## Highlights
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
## Hi @aubinleyoung 🟢 🟢 🟢 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 👏🏼 👏🏼 👏🏼
 
## Congratulations! 💯 💯 💯 

giphy

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Excelent work implementing changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**